Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -88,5 +89,27 @@ protected override void UpdateButtonData(MixedRealityInteractionMapping interact
                 }
             }
         }
+
+        /// <summary>
+        /// Determines whether or not this controller uses MRTK for controller visualization or not
+        /// </summary>
+        public bool UseMRTKControllerVisualization = false;
+
+        /// <inheritdoc/>
+        /// <remarks>
+        /// If UseMRTKControllerVisualization is fals, Oculus Touch controller model visualization will not be managed by MRTK
+        /// Ensure that the Oculus Integration Package is installed and the Ovr Camera Rig is set correctly in the Oculus XRSDK Device Manager
+        /// </remarks>
+        protected override bool TryRenderControllerModel(Type controllerType, InputSourceType inputSourceType)
+        {
+            if (UseMRTKControllerVisualization)
+            {
+                return base.TryRenderControllerModel(controllerType, inputSourceType);
+            }
+            else
+            {
+                return false;
+            }
+        }
```

Comment:
Is this something that needs to be explicitly managed, or could the controller be added to the visualization list with a `null` model? Feels like this might be able to be managed by https://github.com/microsoft/MixedRealityToolkit-Unity/blob/284e501849372398d3167104456be781cb0dd84c/Assets/MRTK/Core/Providers/BaseController.cs#L285-L289

Reply:
Answered my own question! It tries to load the global model if none is provided... :(